package.json

```diff
@@ -55,6 +55,7 @@
     "dayjs": "^1.8.24",
     "i18next": "^19.3.3",
     "lodash.isequal": "^4.5.0",
+    "node-fetch": "^2.6.1",
```

Comment:
I'm a little curious on why we are pulling this library but not using it, did we forget a commit on this PR?

Reply:
trying to get the security vulnerability patch in without having to update all the other upstream libraries that depend on this one.  so being a bit lazy and you're right, we should do this the proper way. Ill close this and cut a card for the work.

Reply:
I believe this is just forcing node-fetch to update in response to the recent vulnerability in it, without updating react-native entirely.